addons/delivery/delivery.py

```diff
@@ -48,135 +46,130 @@ def get_price(self, cr, uid, ids, field_name, arg=None, context=None):
     _columns = {
         'name': fields.char('Delivery Method', required=True, translate=True),
         'sequence': fields.integer('Sequence', help="Determine the display order"),
-        'partner_id': fields.many2one('res.partner', 'Transport Company', required=True, help="The partner that is doing the delivery service."),
+        'partner_id': fields.many2one('res.partner', 'Transporter Company', required=True, help="The partner that is doing the delivery service."),
```

Comment:
Transport company should not be under the Pricing tab but under the Name of the Delivery Method. 

Mockup has been updated according to it.